### ISSUE TITLE
keras cannot save Batchnormalization's axis attiribute.

### ISSUE THREAD

**Issue by hafssol** [fchollet/keras#1517]
I applied BN to CONV2D layer. So I applied axis =1,
After train is over, I saved the model to json string and parameter to h5d file.
But the json string didn't included the axis information of BN.
So I failed to load "h5d file's parameter about BN" to "loaded model from json file"

**Comment by fchollet** [comment]
Fixed.
